### DISCUSSION TITLE
[Linter] Add lint rule for theme colours

### DISCUSSION THREAD

**Discussion by Svish** [tailwindlabs/tailwindcss#11445] [Ideas]
When doing a design with both light (default) and dark (`dark:`) mode, it's very easy to forget to add colours for the mode you're not currently developing in.

Would it be an idea to have an optional lint rule that warns you whenever you add a colour class to an element, without adding the same colour for the alternate theme?

For example if I add `text-gray-800`, then I should most likely also add `dark:text-some-alternative`, and it would be great with a warning/reminder of that.

**Comment by camsong** [comment]
+1 for this, it would be very helpful for team collaboration.
